docs/man/Makefile.am

```diff
@@ -669,6 +669,17 @@ endif
 
 HTML_LINUX_I2C_MANS = asem.html pijuice.html
 
+SRC_LINUX_GPIO_PAGES = gpio.txt
```

Comment:
Just in case: is it right to call this a LINUX_GPIO? The currently used libgpiod "backend" does say it is for linux, but original discussion implied this driver could be more generic with other platforms logically supporting the interface.